regtests/ww3_tp2.22/input/ww3_multi.inp

```diff
@@ -0,0 +1,31 @@
+$ WAVEWATCH III multi-scale input file
+$ ------------------------------
+  1 0 F 1 F F
+$
+  'ww3' 'no' 'no' 'no' 'no' 'no' 'no' 'no' 'no' 'no'   1  1  0.00 1.00  F
+$
+   19680606 000000 19680618 000000
+$
+   T  T
+$
+   19680606 000000  86400  19680618 000000
+$
+ N
+ HS T0M1 DIR SPR
```

Comment:
Add Stoke's drift variables to ensure the new test is properly testing and showing capabilities of new features.

Reply:
I don't see a multi_grid test added to matrix.base, so perhaps removing the multi_related files as these aren't needed?